ci-operator/step-registry/cucushift/installer/rehearse/gcp/cases/clusters/provision/cucushift-installer-rehearse-gcp-cases-clusters-provision-commands.sh

```
@@ -176,6 +190,186 @@ sshKey: |
+
+  if [[ "${machine_type}" == "a3-highgpu-1g" ]]; then
+    echo "$(date -u --rfc-3339=seconds) - Patching preemptible to true for a3-highgpu-1g"
+    yq-v4 eval -i '.spec.template.spec.providerSpec.value.preemptible = true' "${target_ms}"
+  fi
+
+  echo "$(date -u --rfc-3339=seconds) - Debug: Patched providerSpec for ${target_ms}:"
+  yq-v4 eval '.spec.template.spec.providerSpec.value' "${target_ms}"
+
+  # Adjust other MachineSets to maintain IC_COMPUTE_NODE_COUNT total workers
+  local current_total_workers=1
+  for ms in $(ls "${install_dir}/openshift/99_openshift-cluster-api_worker-machineset-"*.yaml | sort); do
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)